web/src/main/java/com/graphhopper/http/GHJerseyViolationExceptionMapper.java

```diff
@@ -19,17 +19,25 @@
 package com.graphhopper.http;
 
 import com.graphhopper.http.api.JsonErrorEntity;
+import com.graphhopper.util.Helper;
 import io.dropwizard.jersey.validation.ConstraintMessage;
 import io.dropwizard.jersey.validation.JerseyViolationException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
 import javax.ws.rs.ext.ExceptionMapper;
+import javax.ws.rs.ext.Provider;
 import java.util.stream.Collectors;
 
+@Provider
 public class GHJerseyViolationExceptionMapper implements ExceptionMapper<JerseyViolationException> {
```

Comment:
btw there is also `io.dropwizard.jersey.validation.JerseyViolationExceptionMapper`, but no idea really :)

Reply:
Good question - does not look too different except now the logging info vs. debug: https://github.com/dropwizard/dropwizard/blob/bb6209d053995c4b2111a0525ae231902f662edb/dropwizard-jersey/src/main/java/io/dropwizard/jersey/validation/JerseyViolationExceptionMapper.java